src/components/signInUp/SignInUp.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./index.css";
+import SignIn from "./sections/SignIn";
+import SignUp from "./sections/SignUp";
+import { useState } from "react";
+const SignInUp = () => {
+  const [logging, setLogging] = useState("signing-in");
```

Comment:
for string constants like these, we usually make them all caps

"SIGNING_IN"
"SIGNING_UP"

change here and its usages